src/wp-admin/edit-form-blocks.php

```diff
@@ -49,20 +49,21 @@ static function( $classes ) {
 wp_enqueue_script( 'heartbeat' );
 wp_enqueue_script( 'wp-edit-post' );
 
-$rest_base = ! empty( $post_type_object->rest_base ) ? $post_type_object->rest_base : $post_type_object->name;
+$rest_base      = ! empty( $post_type_object->rest_base ) ? $post_type_object->rest_base : $post_type_object->name;
+$rest_namespace = ! empty( $post_type_object->rest_namespace ) ? $post_type_object->rest_namespace : 'wp/v2';
 
 // Preload common data.
 $preload_paths = array(
 	'/',
 	'/wp/v2/types?context=edit',
 	'/wp/v2/taxonomies?per_page=-1&context=edit',
 	'/wp/v2/themes?status=active',
-	sprintf( '/wp/v2/%s/%s?context=edit', $rest_base, $post->ID ),
+	sprintf( '/%s/%s/%s?context=edit', $rest_namespace, $rest_base, $post->ID ),
```

Comment:
Let's switch this to use `rest_get_route_for_post` while we're here

Reply:
Does that contain the rest_url as well?